### ISSUE TITLE
Datepicker month/year selection not announced w/ screen-reader

### ISSUE THREAD

**Issue by jelbourn** [angular/material2#11128]
### Summary
Choose month & year range 2016 - 2039 is not being announced in calendar available under basic calendar

### Reproduction on material.angular.io
1. Linear navigate to ' Components ' menu item and click 'Enter' to open components page.                                                                                                                                          
2. Linear navigate forward and set focus on expand collapse left hand menu item ' Form Controls'                                                                                                                               
3. Click 'Enter' key to expand OR collapse this menu item                                                                                                                                                                                                                 
4. Linear navigate into "Datepicker" and click ENTER keystroke.                                                                            
5. Navigate into tabs "Overview" and Click ENTER                                                                                                                                                                                                                             
6. Navigate into "Basic datepicker" heading and navigate into calendar button and press enter button.                                                                                                                
7. Navigate into 'Choose month & year" and click enter button

### Additional comments
Observed on other datepicker components as well.
